src/lib.rs

```diff
@@ -84,6 +84,7 @@ mod bezpath;
 mod circle;
 pub mod common;
 mod cubicbez;
+mod curve_intersections;
```

Comment:
Now we've got the algorithm down, do we actually have a public interface to curve intersections? It looks like I forgot to add one altogether. Should we just expose this module and its function, or add it as a method on the trait, or...?

Reply:
I'm ok with this being a naked function.